addons/binding/org.openhab.binding.enocean/ESH-INF/thing/tcpbridge.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="enocean"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- EnOcean Bridge -->
+	<bridge-type id="tcpbridge">
+		<label>EnOcean TCP Bridge</label>
+		<description>The EnOcean bridge connects over TCP/IP to an ESP3 device.</description>
+
+		<channels>
+			<channel id="repeaterMode" typeId="repeaterMode" />
+			<channel typeId="setBaseId" id="setBaseId" />
+		</channels>
+
+		<properties>
```

Comment:
remove those - properties should only be listed with static values here.

Reply:
removed all properties